sdk/src/test/java/com/silanis/esl/sdk/internal/converter/DocumentPackageSettingsConverterTest.java

```diff
@@ -70,7 +64,7 @@ public void convertNullAPIToAPI() {
     public void convertSDKToSDK() {
 
         sdkPackageSettings1 = createTypicalSDKPackageSettings();
-        sdkPackageSettings2 = new DocumentPackageSettingsConverter(sdkPackageSettings1).toSDKPackageSettings();
+        DocumentPackageSettings sdkPackageSettings2 = new DocumentPackageSettingsConverter(sdkPackageSettings1).toSDKPackageSettings();
```

Comment:
This section doesn't need to be modified.

Reply:
these references are used only once withing the method. they need not be class members.

Reply:
It is correct that it doesn't need to be a class variable.
But for some reasons, this correction is not necessary.

Your commits are completely irrelevant.
You only modified apiPackageSettings2. apiPackageSettings1 does not have to be a class variable either. I wonder why suddenly you just modified this.
This is not the only issue with It is not that only the DocumentPackageSettingsConverterTest has the issue. we need to modify all ConverterTest, if we decide to change.
As you can see, this is a unit test, not a product. It doesn't make much sense to spend time fixing unit tests. And this is not such an important issue at this time.